spring-integration-smb/src/main/java/org/springframework/integration/smb/session/SmbConfig.java

```diff
@@ -52,6 +53,12 @@
 
 	private boolean useTempFile = false;
 
+	/** follows jCIFS library defaults */
```

Comment:
Please, make it as a plain comment (single asterisk or make it as a true JavaDocs:
```suggestion
	/** 
	 * follows jCIFS library defaults 
       */
```

Reply:
Done!